css/style.scss

```diff
@@ -0,0 +1,336 @@
+        height: 16px;
+        border-radius: 10px;
+
+        &:nth-child(1) {
+            background-color: #00C700;
+        }
+        &:nth-child(2) {
+            background-color:#C78F00;
+            ;
+        }
+        &:nth-child(3) {
+            background-color:#C70000;
+        }
+    }
+}
+
+main {
+    display: grid;
+    width: 100%;
+    grid-template-areas:
+        "area1 area2 area4"
+        "area1 area3 area5";
+    grid-template-columns: 1fr minmax(400px, 900px) 1fr;
+    
+    hr {
+        display: none;
+        opacity: 0.05;
+        margin-top: 50px;
+    }
+    #empty-left {
+        grid-area: area1;
+        background-color: $color3;
+    }
+    #main-content {
+        grid-area: area2;
+        background-color: $color1;
+        padding: 24px;
+
+        & > div {
+            display: flex;
+            align-items: center;
+            column-gap: 28px;
+            position: relative;
+            left: -130px;
+
+            h1 {
+                font: 900 40px "Montserrat", sans-serif;
+               }
+        }
+
+        #context {
+            display: flex;
+            align-items:normal;
+            column-gap: 28px;
+            padding-top: 50px;
+            p {
+            
+                font-size: 15px;
+            }
+        }
+
+        article {
+            display: flex;
+            align-items: center;
+            column-gap: 50px;
+            padding-top: 50px;
+            p {
+                font-size: 22px;
+            }
+        }
+
+        
+        
+    }
+    #donate-content {
+        grid-area: area3;
+        background-color: $color2;
+        padding: 24px;
+        display: flex;
+        align-items: center;
+        column-gap: 50px;
+        font-size: 22px;
+        
+        
+        p {
+            padding:28px 0 ;
+        }
+        #doacao {
+            font-family: "Montserrat", sans-serif;
+            padding-bottom: 28px;
+        }
+        div a {
+            display: flex;
+            align-items: center;
+            column-gap: 15px;
+        }
+    }
+        
+        #empty-right-main {
+            grid-area: area4;
+            background-color: $color1;
+        }
+        #empty-right-donate {
+            grid-area: area5;
+            background-color: $color2;
+        }
+
+    span {
+        font: 900 30px "Montserrat", sans-serif;
+    }
+}
+
+.desabilitado {
+    cursor: default;
+
+}
+
+.links-social {
+    column-gap: 5px;
+    display: flex;
+    align-items: center;
+    
+}
+
+a {
+    font-size: 22px;
+    font-family: "Lato", sans-serif;
+    text-decoration: none;
+    color:white;
+    
+}
+
+footer {
+    font-family:"Montserrat" sans-serif;
+    display: flex;
+    flex-direction: column;
+    row-gap: 20px;
+    background-color: $color4;
+    height: 200px;
+    width: 100%;
+    padding:50px 0 200px 25vw;
+
+    a img {
+        padding-right: 10px;
+    }
+}
+ 
+
+ a:hover:not(.desabilitado) {
```

Comment:
Pq esse `a:hover:not(.desabilitado)` não está dentro do bloco `a`? Poderia muito bem ser `&:hover:not(.desabilitado)`. Quanto ao `.disabilitado` todas as classes estão em ingles, essa está em português. É um erro muito comum misturarmos idiomas no código. Mas devemos escolher um idioma no início e mantê-lo.